pkgdown/_pkgdown.yml

```diff
@@ -9,6 +9,8 @@ reference:
     contents:
       - geom_text_repel
       - geom_label_repel
+      - position_nudge_repel
+      - position_nudge_label
```

Comment:
I think this line should be deleted.

Reply:
I think position_nudge_repel should remain in the file as there is a new help page generated for it, while I have deleted position_nudge_label which was left behind forgotten from some playing around...